pkg/process/exports/check_outputs.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package exports
```

Comment:
What do you think about having it in the `checks` package instead?

Reply:
I think we can do that. I think in the future we will want to refactor exported variables out of our main package and put them in an exports package, but for now there isn't really any need.

Reply:
Ah interesting, so this would be the place for all `expvars`? Makes sense.